### PR TITLE
EXP-2336 fix resolve to null observable

### PR DESCRIPTION
**Pull request by cesar-oyarzun-m**

### CODE REVIEW

File: exp-android-sdk/src/main/java/com/scala/exp/android/sdk/model/Device.java [hunk 1]
```diff
@@ -42,13 +42,13 @@ public void setExperience(Experience experience) {
         this.experience = experience;
     }
 
-    public static ExpObservable<Device> getCurrentDevice(){
+    public static ExpObservable<?> getCurrentDevice(){
```
Comment (adamgallowayscala) [inline exp-android-sdk/src/main/java/com/scala/exp/android/sdk/model/Device.java]: the generics here should stay Device

Comment (cesar-oyarzun-m) [inline exp-android-sdk/src/main/java/com/scala/exp/android/sdk/model/Device.java]: If I want to return null I need to use generic

Comment (cesar-oyarzun-m) [inline exp-android-sdk/src/main/java/com/scala/exp/android/sdk/model/Device.java]: something like this return new ExpObservable<Device>(Observable.<Device>just(null));

**Comment by cesar-oyarzun-m** [comment]
The third time's the charm ahhahaha!!!

**Comment by adamgallowayscala** [comment]
👍

**Pull request merged**